src/components/Match/matchColumns.jsx

```diff
@@ -561,6 +561,7 @@ export const performanceColumns = [
     sortFn: true,
     displayFn: (row, col, field) => formatSeconds(field) || '-',
     relativeBars: true,
+    inverseRelativeBars: true,
```

Comment:
can you rename this to "invertBarColor" which is more descriptive?

Reply:
Done